<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInDialog</class>
 <widget class="QDialog" name="LogInDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/buttons.qrc">
    <normaloff>:/img_ico/airlines.ico</normaloff>:/img_ico/airlines.ico</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>410</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="upperImageLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>540</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img_background/loginBackground_3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#exitButton{border-image: url(:/img_normal/exit.png);}
QPushButton#exitButton:hover{border-image: url(:/img_hover/exit_hover.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>540</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 252, 255);</string>
    </property>
    <widget class="QCheckBox" name="managerCheckBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>260</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{color: rgb(180, 180, 180);}
QCheckBox:hover{color:rgb(100,100,100);}
QCheckBox:checked{color:rgb(100,100,100);}</string>
     </property>
     <property name="text">
      <string>I'm a manager</string>
     </property>
    </widget>
    <widget class="QLabel" name="headLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img_background/no_head.png);</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>240</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#loginButton{border-radius:4px;
background-color:/*#44cef6;*/
#2ca9e1;
color:#ffffff}
QPushButton#loginButton:hover{background-color:#44cef6;}
QPushButton#loginButton:pressed{background-color:#007bbb}</string>
     </property>
     <property name="text">
      <string>登   录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userNameLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>240</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#userNameLineEdit:hover{border: 1px solid blue}
QLineEdit#userNameLineEdit{
border-top-left-radius:3px;
border-top-right-radius:3px;
border-left:1px solid rgb(200, 200, 200);
border-right:1px solid  rgb(200, 200, 200);;

border-top:1px solid  rgb(200, 200, 200);
background:#ffffff;
padding:5px;}</string>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>240</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#passwordLineEdit{border-bottom-left-radius:3px;;
border-bottom-right-radius:3px;
border-left:1px solid rgb(200, 200, 200);;
border-right:1px solid rgb(200, 200, 200);;
border-bottom:1px solid rgb(200, 200, 200);;
border-top:1px solid rgb(200, 200, 200);
background:#ffffff;
padding:5px}
QLineEdit#passwordLineEdit:hover
{border:1px solid blue}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="helpButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>185</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#helpButton{
	border-image: url(:/img_normal/help_normal.png);}
QPushButton#helpButton{
	
	border-image: url(:/img_hover/help_hover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;注册&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#registerButton{border-image: url(:/img_normal/register.png);}
QPushButton#registerButton:hover{border-image: url(:/img_hover/register_hover.png);}
QPushButton#registerButton:pressed{border-image: url(:/img_normal/register.png);}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="nameWarningLabel">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>25</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img_normal/warning.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="passwordWarningLabel">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>65</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img_normal/warning.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="headSelectButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线 Light</family>
       <pointsize>10</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#headSelectButton{border:none
;color:#2ca9e1;background-color:rgb(235,251,255)}QPushButton#headSelectButton:hover{color:#44cef6;}
QPushButton#headSelectButton:pressed{color:#007bbb}</string>
     </property>
     <property name="text">
      <string>选择头像</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userNameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>managerCheckBox</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources>
  <include location="Resources/buttons.qrc"/>
 </resources>
 <connections/>
</ui>
